src/components/Loader/Loader.scss

```diff
@@ -0,0 +1,84 @@
+@import '../../variables.scss';
+
+.loader {
+  background: linear-gradient(270deg, #000000, #19d4ac);
+  background-size: 400% 400%;
+  height: 100%;
+  padding-top: calc(25% - 50px);
+  width: 100%;
+  -webkit-animation: gradient-animation 10s ease infinite;
+  animation: gradient-animation 10s ease infinite;
+}
+
+@-webkit-keyframes gradient-animation {
+    0%{background-position:0% 50%}
+    50%{background-position:100% 50%}
+    100%{background-position:0% 50%}
+}
+@keyframes gradient-animation {
```

Comment:
This loader looks incredible!  I'm really impressed looking through the css for it!